src/Containers/LogoListing/Logo.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '../../Inputs/Button/Button';
+import { Card } from '../Card/Card';
+
+export interface ILogoProps {
+    listingTitle: string;
+    titleSummary: string;
```

Comment:
maybe we can rename things to

title: string
description: string

to keep the names clear